apps/backend/app/repositories/organization_repository.py

```diff
@@ -177,8 +145,37 @@ async def get_pdf_as_link_by_recipient(recipient_email: str) -> bool:
                     )
                 )
             )
-            result = await session.execute(query)
-            pdf_as_link = result.scalar()
-            if pdf_as_link is None:
-                return True
-            return pdf_as_link
+            value = result.scalar()
+            return value if value is not None else True
```

Comment:
```suggestion
            link = result.scalar()
            return link if link is not None else True
```

the naming was better before 👀 
why are you returining `True` if value is `None`? also - this is why it's better to always check for explicit conditions: if somewhere higher in the call stack of this function, you check for example for `if value` or `if not value`, since you return `True` here, it would treat it as if everything worked just fine (which didn't, because you're returning `True` here when it was `None`). this can be annoying to debug